### PR TITLE
Add testcase for load and store instructions

### PR DESCRIPTION
**Pull request by joii2020**
vleX.v
vseX.v
vlseX.v
vsseX.v
vluxeiX.v
vsuxeiX.v

### CODE REVIEW

File: cases/Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ num-traits = { version = "0.2.14", default-features = false }
 rand = { version = "0.8.4", default-features = false }
 # uint = { version = "0.9.1", default-features = false }
 rvv-simulator-runtime = { git = "https://github.com/TheWaWaR/rvv-prototype.git", rev="52a7a42" }
-
+eint = { git="https://github.com/joii2020/eint.git", branch="master" }
```
Comment (XuJiandong) [inline cases/Cargo.toml]: use official one

File: cases/src/narrowing_integer_right_shift_cases.rs [hunk 2]
```diff
@@ -53,7 +53,7 @@ pub fn test_narrowing_integer_right_shift() {
         });
     }
     let sew = 256u64;
-    for lmul in [-2, 1, 4] {
+    for lmul in [-2, 1, 2] {
```
Comment (XuJiandong) [inline cases/src/narrowing_integer_right_shift_cases.rs:57]: why is the range is changed?

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: cases/src/load_store_cases.rs [hunk 3]
```diff
@@ -58,107 +131,758 @@ fn test_stride(sew: usize, stride: usize) {
...
+            result2,
+            expected2,
+            offset,
+            mem
+        );
+        panic!("Abort");
+    }
+}
+
+fn test_indexed_ordered(sew: usize, offset_sew: usize, lmul: i64) {
```
Comment (XuJiandong) [inline cases/src/load_store_cases.rs]: why doesn't the ordered and unordered use different versions?

File: cases/src/load_store_cases.rs [hunk 4]
```diff
@@ -58,107 +131,758 @@ fn test_stride(sew: usize, stride: usize) {
...
+                let mut buf = buf.as_mut_slice();
+                match offset_sew {
+                    8 => buf.write_u8(offset_sew, i, E8::from(index as u64)),
+                    16 => buf.write_u16(offset_sew, i, E16::from(index as u64)),
+                    32 => buf.write_u32(offset_sew, i, E32::from(index as u64)),
+                    64 => buf.write_u64(offset_sew, i, E64::from(index as u64)),
+                    128 => buf.write_u128(offset_sew, i, E128::from(index as u64)),
+                    256 => buf.write_u256(offset_sew, i, E256::from(index as u64)),
+                    512 => buf.write_u512(offset_sew, i, E512::from(index as u64)),
+                    1024 => buf.write_u1024(offset_sew, i, E1024::from(index as u64)),
```
Comment (XuJiandong) [inline cases/src/load_store_cases.rs]: offset_sew can't be larger than 64.

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: cases/Cargo.toml (hunk 1 above)
Comment (joii2020) [reply cases/Cargo.toml]: The original eint depends on std, and i will send e PR to eint.

File: cases/src/load_store_cases.rs (hunk 3 above)
Comment (joii2020) [reply cases/src/load_store_cases.rs]: The instruction has vluxei and vloxei, but their code is the same.

File: cases/src/narrowing_integer_right_shift_cases.rs (hunk 2 above)
Comment (joii2020) [reply cases/src/narrowing_integer_right_shift_cases.rs:57]: Sorry, The version of ckb-vm I used for debugging is a bit low, and it can be updated after.

**Review by XuJiandong — APPROVED** [review]

**Pull request merged**